pdns/dnspacket.hh

```diff
@@ -121,6 +122,9 @@ public:
   bool couldBeCached() const; //!< returns 0 if this query should bypass the packet cache
   bool hasEDNSSubnet() const;
   bool hasEDNS() const;
+  bool hasEDNSCookie() const;
+  bool hasWellFormedEDNSCookie() const;
+  bool hasValidEDNSCookie(); // Not const, some cookie params might be set
```

Comment:
I see nothing being set in any called code, so I put the `const` back (also in the .cc) and it compiles. I suspect the comment was true for a previous version :-)